app/Traits/GetInvoices.php

```diff
@@ -59,24 +51,14 @@ public function getWeeklyInvoices(Admin $admin) {
 	// Return array of integers of this month's invoices
 	public function getMonthlyInvoices(Admin $admin) {
 
-		$invoices_array = [];
-		$invoices = $admin->invoices;
-		$now = Carbon::now();
-		$startOfMonth = $now->startOfMonth();
-
-		for($i = 1; $i <= $now->daysInMonth; $i++) {
-			$invoices_array[$i] = 0;
-		}
+		$invoices_array = array_fill(1, now()->daysInMonth, 0);
```

Comment:
Similar to above, just populating the array with 0 --> {days in month} number of days